app/controllers/orders_controller.rb

```diff
@@ -10,6 +10,22 @@ def create
     @order = Order.new(user: current_user, total: current_user.cart.total_cart_price)
     if @order.save
       @order.purchase_line_items
+
+      customer = Stripe::Customer.create(
+        email:  @order.user.email,
+        source: params[:stripeToken]
+      )
+      @order.card_token = customer.id
+
+      amount = @order.total
```

Comment:
undefined local variable or method `order' for #<OrdersController:0x007f967c894d28> Did you mean? @order

:(